packages/react-storage/src/components/StorageBrowser/views/LocationsView/__tests__/useLocationsView.spec.tsx

```diff
@@ -269,7 +269,7 @@ describe('useLocationsView', () => {
       hasError: false,
       isLoading: false,
     };
-    mockUseLocationsData(mockDataState);
+    const handleList = mockUseLocationsData(mockDataState);
```

Comment:
Nit: This isn't really related to your PR but `mockUseLocationsData` is now kind of misleading given that it wraps `useLocationsDataSpy` which is actually spying `useListLocations` 😵

Reply:
yeah, we should really avoid creating these test utils